### DISCUSSION TITLE
Rsi la1 and la2

### DISCUSSION THREAD

**Discussion by davnr** [iptv-org/community#459] [Q&A] [answered]
Please, can you provide me a valid link to watch rsi la1 and rsi la2? Thank you very much

**Comment by HNJAMeindersma** [comment, marked as answer]
You can [search](https://iptv-org.github.io/) the iptv-org database yourself.

[`RSI La 1`](https://iptv-org.github.io/channel?id=RSILa1.ch) and [`RSI La 2`](https://iptv-org.github.io/channel?id=RSILa2.ch) do not seem to offer a publicly available stream right now.
